build.bat

```diff
@@ -1,5 +1,4 @@
 @echo off
-setlocal
```

Comment:
What was it used for?

Reply:
To every variable set here do not "leak" to console, but removed in order to vcvars64 not have to be called everytime.